dts/bindings/gpio/rohm,bd8lb600fs-gpio.yaml

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2022 SILA Embedded Solutions GmbH
+# Copyright (c) 2024 SILA Embedded Solutions GmbH
```

Comment:
Add the new year without removing the previous one

Reply:
done